.github/workflows/release.yaml

```diff
@@ -20,8 +20,17 @@ jobs :
           java-version : '11'
           check-latest : true
 
-      - name : Publish Release
-        run : ./gradlew clean publish --no-build-cache --no-daemon --stacktrace --no-parallel && cd gradle-plugin && ./gradlew clean publish --no-build-cache --no-daemon --stacktrace && cd ..
+      # TODO: Revert release changes when we drop support for 1.7
+      - name : Publish Release 1.7
+        run : ./gradlew clean publish --no-build-cache --no-daemon --stacktrace --no-parallel -Psquare.kotlinVersion=1.7.20 && cd gradle-plugin && ./gradlew clean publish --no-build-cache --no-daemon --stacktrace -Psquare.kotlinVersion=1.7.20 && cd ..
+        env :
+          ORG_GRADLE_PROJECT_mavenCentralUsername : ${{ secrets.SONATYPE_NEXUS_USERNAME }}
+          ORG_GRADLE_PROJECT_mavenCentralPassword : ${{ secrets.SONATYPE_NEXUS_PASSWORD }}
+          ORG_GRADLE_PROJECT_signingInMemoryKey : ${{ secrets.ARTIFACT_SIGNING_PRIVATE_KEY }}
+          ORG_GRADLE_PROJECT_signingInMemoryKeyPassword : ''
+
+      - name : Publish Release 1.8
+        run : ./gradlew clean publish --no-build-cache --no-daemon --stacktrace --no-parallel -Psquare.kotlinVersion=1.8.0-RC -PVERSION_NAME=2.4.3-1-8-SNAPSHOT && cd gradle-plugin && ./gradlew clean publish --no-build-cache --no-daemon --stacktrace -Psquare.kotlinVersion=1.8.0-RC -PVERSION_NAME=2.4.3-1-8-SNAPSHOT && cd ..
```

Comment:
`-PVERSION_NAME=2.4.3-1-8-SNAPSHOT` looks wrong here. This is not a snapshot.

Reply:
🤦